azure-pipelines.yml

```diff
@@ -0,0 +1,72 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+variables:
+  imageName: 'manage-courses-support-poc'
+  # Support multiple docker-compose files to allow travis and az pipelines to co exist: https://docs.docker.com/compose/extends/#multiple-compose-files
+  dockerOverride: 'docker-compose -f docker-compose.yaml -f docker-compose.az.yml'
+
+steps:
+- script: |
+    GIT_SHORT_SHA=$(echo $(Build.SourceVersion) | cut -c 1-7)
+    docker_path=$(dockerHubUsername)/$(imageName)
+    echo "##vso[build.updatebuildnumber]$GIT_SHORT_SHA"
+    echo "##vso[task.setvariable variable=docker_path;]$docker_path"
+  displayName: 'Set version number'
+
+- script: |
+    $DOCKER_OVERRIDE build
+    $DOCKER_OVERRIDE up --no-build -d
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c "./wait-for-it.sh db:5432"
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec rails db:setup"
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec rake assets:precompile"
```

Comment:
Does `assets:precompile` require the db? If not, how about we do it before the `wait-for-it.sh` to make better use of time?

Reply:
Lets check with @benilovj to make sure it doesn't break anything.